README.md

```diff
@@ -169,6 +169,129 @@ docker inspect docker-beegfs-demo_beegfs
 
 Now you can connect to the management node from another machine using its ip address.
 
+## Setup on non-virtual (physical) machine (Ubuntu 22.04) && Potential troubleshooting:
+At first, you should follow previously mentioned instructions. Whether there are problems still, you can try the following fixes.
+
+There are few tricky moments that aren’t mentioned here and very poorly mentioned on the internet. The thing is, the physical machine with docker **MUST** be connected via ethernet. Whether it is connected via wifi, it is likely that the addresses of management/metadata/storage nodes couldn’t be pinged.
```

Comment:
Let's add to `docker-compose.yml` comment that wifi interfaces are not suitable for `macvlan`.